src/viewer/entities/playerVehicle.ts

```diff
@@ -59,6 +63,12 @@ class PlayerVehicle extends Entity {
 			const speed = addCommas(Math.floor(msToKnots(this.velocity.length())));
 			this.textOverlay.edit(`${this.owner.pilotName} [${this.displayName}]\n${Math.floor(mToFt(this.position.y))}ft\n${speed}kn`);
 		}
+
+		if (this.app.isOfflineTest && this.lastTestingCMSDispense + 6000 < Date.now()) {
```

Comment:
Testing code like this should go into the offlineTestSetup, you can use `setInterval` to create a loop that calls the fireCm method at some rate.